### PR TITLE
homebrew tap for spice and spiced

### PR DESCRIPTION
**Pull request by y-f-u**
2 taps to install spice and spiced.

Need to make a further change `spice`'s assumption on spiced exists in ~/.spice/bin.

Now even after installing both, run `spice run` will pull another copy of spiced into ~/.spice/bin

### CODE REVIEW

File: Formula/spice.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  if OS.mac?
+    if Hardware::CPU.arm?
+        url "https://github.com/spiceai/spiceai/releases/download/#{version}/spice_darwin_aarch64.tar.gz"
+        sha256 "2ba8a309b80dba160f56f938962bca0014e8414fe9e635a600175b155cf2004a"
+    else
+        url "https://github.com/spiceai/spiceai/releases/download/#{version}/spice_darwin_x86_64.tar.gz"
+        sha256 "28227e7f02be240a5d96c609a472b6960cd67e7136861a0096bd42df86346ffd"
+    end
+  elsif OS.linux?
+    if Hardware::CPU.arm?
```
Comment (y-f-u) [inline Formula/spice.rb:20]: brew can't run on arm yet + we don't have linux arm build

Comment (y-f-u) [reply Formula/spice.rb:20]: leave it for future.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Install Spice from Brew
+
+## Pre-requisites
+
+Noto: when using linux homebrew, install build-essentials or equivalent package.
+
+## Install `spice`
+```
+brew install spiceai/spiceai/spice
```
Comment (lukekim) [inline README.md:9]: It's not possible to make this root, is it? So we can do `brew install spiceai/spiceai`?

File: Formula/spice.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    else
+        url "https://github.com/spiceai/spiceai/releases/download/#{version}/spice_linux_x86_64.tar.gz"
+        sha256 "eaf89b7ef221837091bd8a6a435401fec12c4d249bc0474cceefeac9c64630e2"
+    end
+  end
+
+  def install
+      bin.install "spice" => "spice"
```
Comment (lukekim) [inline Formula/spice.rb:28]: Should probably be `spiceai` at least for now.

File: README.md (hunk 2 above)
Comment (y-f-u) [reply README.md:9]: yeah would be nice if they allow so. But I haven't seen any doc about this.

**Pull request merged**